Week-1/Assignment-1.2/main.css

```diff
@@ -0,0 +1,142 @@
+* {
+    margin: 0;
+    padding: 0;
```

Comment:
Here you can also add box-sizing as border-box so that your width and height are always inclusive of padding and border.